.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: 'Report an unexpected behavior '
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Where are you using Cairo Native**
+In which project are you using Cairo Native
+
+**Version**
+What Cairo Native commit are you using
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+* What project are you running
+* Which Cairo, Sierra, or Starknet transaction are you running
+* What Error are you getting
```

Comment:
Could we also request the exact commands for reproducing the bug as well as a minimal example in cases where it makes sense? (maybe a contract is impossible to reduce but if it's a libfunc there should be a minimal example).

Reply:
Sure!
Good advice @azteca1998